packages/esm-form-entry-app/src/config-schema.ts

```diff
@@ -0,0 +1,17 @@
+import { Type } from '@openmrs/esm-framework';
+
+export const configSchema = {
+  dataSources: {
+    monthlySchedule: {
+      _type: Type.Boolean,
+      _default: false,
+      _description:
+        'Whether to use monthly scheduled appointment data source in form-entry engine. Requires `baseEtlUrl`.',
+    },
+  },
+  baseEtlUrl: {
```

Comment:
Please change name of baseETLUrl to "appointmentsResourceUrl" and then this can be merged

Reply:
In that case @donaldkibet  you'll also want to change the `_default` to point directly at the appointments resource, instead of being the base URL, and change the usages appropriately.